optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/anno/entity/TestdataAnnoEntitySubChangeValRange.java

```diff
@@ -0,0 +1,63 @@
+package org.optaplanner.core.impl.testdata.domain.anno.entity;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.solution.PlanningEntityProperty;
+import org.optaplanner.core.api.domain.solution.PlanningScore;
+import org.optaplanner.core.api.domain.solution.PlanningSolution;
+import org.optaplanner.core.api.domain.valuerange.ValueRangeProvider;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.api.score.buildin.simple.SimpleScore;
+import org.optaplanner.core.impl.domain.entity.descriptor.EntityDescriptor;
+import org.optaplanner.core.impl.domain.solution.descriptor.SolutionDescriptor;
+import org.optaplanner.core.impl.domain.variable.descriptor.GenuineVariableDescriptor;
+
+@PlanningEntity
+public class TestdataAnnoEntitySubChangeValRange extends TestdataAnnoEntityParent {
+
+    @PlanningSolution
+    public static interface Solution {
```

Comment:
A new class also named Solution?

Reply:
Yeah I was just lazy to create a class for the solution. I though an interface would do, especially when Java 8 allows the default method implementations. After thinking twice about it, this actually looks like a kind of anti-pattern to use an interface as a planning solution, so I'm gonna change all of them into classes.